src/deprecated.jl

```diff
@@ -0,0 +1,4 @@
+import Base: @deprecate
```

Comment:
`using` is slightly better than `import` when you don't add methods to the function.